xds/internal/xdsclient/bootstrap/bootstrap_test.go

```diff
@@ -1034,4 +1038,23 @@ func TestCredsBuilders(t *testing.T) {
 	if got, want := i.Name(), "insecure"; got != want {
 		t.Errorf("insecureCredsBuilder.Name = %v, want %v", got, want)
 	}
+
+	tcb := &tlsCredsBuilder{}
+	if _, err := tcb.Build(nil); err == nil {
+		t.Errorf("tlsCredsBuilder.Build succeeded, want failure")
+	}
+	if got, want := tcb.Name(), "tls"; got != want {
+		t.Errorf("tlsCredsBuilder.Name = %v, want %v", got, want)
+	}
+}
+
+func TestTlsCredsBuilder(t *testing.T) {
+	tls := &tlsCredsBuilder{}
+	if _, err := tls.Build(json.RawMessage(`{}`)); err != nil {
+		t.Errorf("unexpected error with empty config: %s", err)
+	}
+	if _, err := tls.Build(json.RawMessage(`{"ca_certificate_file":"/ca_certificates.pem","refresh_interval": "asdf"}`)); err == nil {
+		t.Errorf("expected an error with invalid refresh interval")
```

Comment:
Nit: This and the above failure messages are a little hard to read. How about
- `tls.Build() failed for an empty config`
- `tls.Build() succeeded for a config with invalid refresh interval, when expected to fail`

See: https://google.github.io/styleguide/go/decisions#identify-the-function

Reply:
I took your suggestion. The google style guide goes a bit further and ask for a format with the argument. I didn't follow this because I didn't find it made the output more clear, but please let me know if you'd prefer otherwise.